### PR TITLE
(#21) Search Terms Without Definition

### LINKED ISSUE #21: SWS Results Page Breaks for search terms without definitions
### Issue description
If I enter a search term in the URL on develop that does not have a definition (such as DCEG or NCI Connect), the search breaks and returns the "Error has occurred" page.  I should be getting the results page for that term.

> **ESTIMATE** 20

### Steps to reproduce the issue

1.  Go to https://react-app-dev.cancer.gov/sitewide-search-app/develop?cfg=0&swKeyword=tumor
2. Change swKeyword to DCEG
3. Note the page returns the error has occurred pge.

### What's the expected result?
I should be getting search results for DCEG

### What's the actual result?
The app is breaking right now for any term that is searched that does not have an associated definition.

### Related Tickets
* Issue #9999
* Issue #9999

### PR DESCRIPTION
**Pull request by seyilonge-nci**
Closes #21

* Fixed definition display issue when no results are returned for term
* Fixed results display issue when there's no title for returned payload
* Added integration test for definition box when no results are returned

### CODE REVIEW

**Comment by seyilonge-nci** [comment]
@VictoriaSunNIH 
Please review https://react-app-dev.cancer.gov/sitewide-search-app/bugfix__21-search-without-definition?cfg=0&swkeyword=dceg

File: src/components/molecules/definition/definition.jsx [hunk 1]
```diff
@@ -7,9 +7,9 @@ import { i18n, splitSentencesToArray } from '../../../utils';
 
 const Definition = ({ results }) => {
 	const payload = results[0];
-	const definitionSentencesArray = splitSentencesToArray(
-		payload.definition.html
-	);
+	const definitionSentencesArray = payload
```
Comment (arcepaul) [inline src/components/molecules/definition/definition.jsx:10]: should this be payload && payload.definition ?

Comment (seyilonge-nci) [reply src/components/molecules/definition/definition.jsx:10]: 👍

Comment (seyilonge-nci) [reply src/components/molecules/definition/definition.jsx]: Updated, thanks!

**Pull request merged**